Assets/HotmaxEditorVR/universalTransform.cs

```diff
@@ -0,0 +1,223 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class universalTransform : MonoBehaviour
+{
+    public objectSelect objSelect;
+
+    float initialPadYPosition;
```

Comment:
be explicit with declaring private variables -- also would help to group variables together that are related and add a comment above the group that briefly states functionality

Reply:
do you mean by putting an underscore before? or the actual name?